CONTRIBUTING.md

```diff
@@ -15,7 +19,7 @@ can install ruff with:
 
 Otherwise, you can install ruff in your virtual environment with:
 
-    pip install ruff
+    uv pip install ruff
```

Comment:
If we're going to use the latest version, then I think the uv way would be:

```
uv tool install ruff
# now it's available globally

uvx
# ruff v0.6.5

uvx ruff --help
# blah blah blah
```

Reply:
I'd rather keep the snap as the preferred way since it'll do auto-updates (which `uv tool` and `uvx` don't), but I agree that we should be using

```suggestion
    uv tool install ruff
```

for someone who'd rather use `uv` to install ruff.